submodules.json

```diff
@@ -1,17 +1,28 @@
 [
-    {
-        "name" : "react-boilerplate",
-        "url" : "https://github.com/react-boilerplate/react-boilerplate",
-        "tags" : ["react", "reactjs"]
-    },
-    {
-        "name" : "node-boilerplate",
-        "url" : "https://github.com/hagopj13/node-express-boilerplate",
-        "tags" : ["node", "nodejs"]
-    }
-    {
-        "name": "repo-boilerplate",
-        "url": "https://github.com/mintel/repo-boilerplate",
-        "tags" : ["none"]
-    }
+
```

Comment:
You might've removed a `{` while resolving the merge conflict. Please do keep that.